dashboard/test/integration/curriculum_docs_test.rb

```diff
@@ -188,7 +188,7 @@ def setup
     end
 
     test "environment get_summary_by_name query count" do
-      assert_queries(7) do
+      assert_queries(8) do
```

Comment:
In order to localize programming expressions, it is necessary to add the programming_environment.name in the scope. Which add an additional query in the summarize_for_show method.